apps/examples/testcase/le_tc/kernel/tc_wqueue.c

```diff
@@ -0,0 +1,153 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file wqueue.c
+/// @brief Test Case Example for Work Queue API
+/**************************************************************************
+*
+*   Copyright (C) 2007-2009, 2011 Gregory Nutt. All rights reserved.
```

Comment:
Let's remove Nuttx license from this file.

Reply:
Done